app/views/admin_applications/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Application Details</h2>
+
+<p>Application Status: <%= @app.status %></p>
+<p>Name: <%= @app.name %></p>
+<p>Address: <%= @app.full_address %></p>
+<p>Description: <%= @app.description %></p>
+<h4>Pets</h4>
+<% @app.pets.each do | pet | %>
+    <section id="pet-<%= pet.id %>">
+      <span><%= pet.name %></span>
+      <% if pet.adoptable %>
+          <%= form_with url: "/pets/#{pet.id}?approve=true&app_id=#{@app.id}",
+                        method: :patch,
+                        data: {turbo: false},
+                        html: {style: "display: inline-block"} do |form| %>
+            <% form.submit "Approve"  %>
```

Comment:
Is the html block a sort of form formatting? I haven't seen that before

Reply:
yeah, I wanted everything inline, and stack overflow, master sensei, taught me this one.  without the span tag and the style inline-block then everything would've been stacked vertically